portlets/src/main/webapp/vue-app/realizations/realizationsServices.js

```diff
@@ -1,5 +1,5 @@
-export function getAllRealizations(fromDate, toDate, sortBy, sortDescending, offset, limit) {
-  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=${fromDate}&toDate=${toDate}&sortBy=${sortBy}&sortDescending=${sortDescending}&offset=${offset || 0}&limit=${limit|| 10}`, {
+export function getAllRealizations(fromDate, toDate, earnerId, sortBy, sortDescending, offset, limit, returnType) {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=${fromDate}&toDate=${toDate}&earnerId=${earnerId}&sortBy=${sortBy}&sortDescending=${sortDescending}&offset=${offset || 0}&limit=${limit|| 10}&returnType=${returnType}`, {
```

Comment:
No need to specify the retrunType when it's always `json`

```suggestion
export function getAllRealizations(fromDate, toDate, earnerId, sortBy, sortDescending, offset, limit) {
  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=${fromDate}&toDate=${toDate}&earnerId=${earnerId}&sortBy=${sortBy}&sortDescending=${sortDescending}&offset=${offset || 0}&limit=${limit|| 10}`, {
```

Reply:
done !